packages/ui-extensions/src/surfaces/admin/components/Badge/Badge.doc.ts

```diff
@@ -29,14 +15,12 @@ const data: ReferenceEntityTemplateSchema = {
         },
         {
           title: 'HTML',
-          code: './examples/default.html',
-          language: 'preview-admin-inline',
+          code: './examples/default.html', // This gets updated in build-ab-docs.sh
+          language: 'preview', // Do not edit. This is required for Live Previews
```

Comment:
I added this little shim `preview-admin-inline` to swap between different templates for the Admin live preview. These map to different [template wrappers](https://github.com/Shopify/shopify-dev/blob/a492cea8087ce4e2b5a5fd279c613ba9dd40093d/app/ui/sections/components/LivePreview/LivePreview.tsx#L20-L36). I think we should keep them until this ships https://github.com/Shopify/generate-docs/pull/332